src/js/base/ffi-helpers.js

```diff
@@ -456,6 +456,8 @@ define(["js/runtime-util", "trove/lists", "trove/sets", "trove/option", "trove/e
 
         makeList: makeList,
         makeTreeSet: makeTreeSet,
+
+        isOption: runtime.getField(O, "is-Option"),
```

Comment:
That's not an acceptable reason, in the long term -- it feels like a type error to me, since all other functions are raw JS functions...  (I also don't know which came first, that `ffi.cases` used `app` for us, or the annotations were changed to not include the `app`.)  @jpolitz opinions which way to resolve this?